assets/js/theme/common/carousel/utils/arrowAriaLabling.js

```diff
@@ -1,26 +1,28 @@
-const NUMBER = '[NUMBER]';
-const integerRegExp = /[0-9]+/;
-const lastIntegerRegExp = /(\d+)(?!.*\d)/;
+export const updateTextWithLiveData = (textForChange, firstData, secondData) => {
+    const NUMBER = '[NUMBER]';
+    const integerRegExp = /[0-9]+/;
+    const lastIntegerRegExp = /(\d+)(?!.*\d)/;
```

Comment:
This vars are static and I think we can replace them to not reinit them on each function usage

Reply:
agree